### DISCUSSION TITLE
Module is not working with angular 9

### DISCUSSION THREAD

**Discussion by Bounguicha** [MurhafSousli/ngx-sharebuttons#569] [Q&A] [answered]
Hey im implementing the ngx-sharebutons with Angular 9 and its not working because of much angular/core.
 this module is not compatible with angular 9

**Comment by MurhafSousli** [comment, marked as answer]
If [v8.0.5](https://github.com/MurhafSousli/ngx-sharebuttons/blob/master/CHANGELOG.MD#805) didn't work, try [v8.0.4](https://github.com/MurhafSousli/ngx-sharebuttons/blob/master/CHANGELOG.MD#804)
